file/lib/file_source_endpoint.ex

```diff
@@ -125,28 +155,20 @@ defmodule Membrane.RTC.Engine.Support.FileSourceEndpoint do
     {[notify_parent: track_ready], state}
   end
 
-  defp convert_to_payloader_format(:OPUS) do
-    fn link_builder ->
-      child(link_builder, :decoder, Membrane.AAC.FDK.Decoder)
-      |> child(:encoder, %Membrane.Opus.Encoder{
-        input_stream_format: %Membrane.RawAudio{
-          channels: 1,
-          sample_rate: 48_000,
-          sample_format: :s16le
-        }
-      })
-      |> child(:parser, %Membrane.Opus.Parser{})
-    end
-  end
-
-  defp convert_to_payloader_format(:H264) do
-    fn link_builder ->
-      child(link_builder, :parser, %Membrane.H264.Parser{
-        generate_best_effort_timestamps: %{
-          framerate: {60, 1}
-        },
-        output_alignment: :nalu
-      })
-    end
+  @spec identitiy_transformation(ChildrenSpec.builder()) :: ChildrenSpec.builder()
+  def identitiy_transformation(link_builder), do: link_builder
+
+  @spec transform_aac_to_opus(ChildrenSpec.builder()) :: ChildrenSpec.builder()
+  def transform_aac_to_opus(link_builder) do
+    link_builder
+    |> child(:decoder, Membrane.AAC.FDK.Decoder)
+    |> child(:encoder, %Membrane.Opus.Encoder{
+      input_stream_format: %Membrane.RawAudio{
+        channels: 1,
+        sample_rate: 48_000,
+        sample_format: :s16le
+      }
+    })
+    |> child(:parser, %Membrane.Opus.Parser{})
```

Comment:
I don't know if we want to keep this function (we got rid of the other one), but if so, I think I would put it in another module